app/customer.py

```diff
@@ -0,0 +1,85 @@
+from __future__ import annotations
+import math
+from dataclasses import dataclass
+from app.car import Car
+from datetime import datetime
+from app.shop import Shop
+
+
+class NotEnoughMoney(Exception):
+
+    def __init__(self) -> None:
+        super().__init__()
```

Comment:
There is no need to have this `__init__`, it does not add any additional logic and you can leave class with `pass` only.